src/Vlingo.Xoom/actors/Settings.cs

```diff
@@ -11,34 +11,33 @@
 using System.Linq;
 using System.Reflection;
 
-namespace io.vlingo.xoom.actors {
+namespace Vlingo.Xoom.Actors {
     public class Settings {
 
-        private static Dictionary<string, string> PROPERTIES = new Dictionary<string, string>();
-        private static readonly string PROPERTIES_FILENAME = "/vlingo-xoom.properties";
-        private static readonly long serialVersionUID = 1L;
-        private static readonly Dictionary<object, object> DEFAULT_DATABASE_PROPERTIES = new Dictionary<object, object>() {
+        private static Dictionary<string, string> properties = new Dictionary<string, string>();
+        private static string propertiesFileName = "/vlingo-xoom.properties";
```

Comment:
This is an old format not longer used on .NET platform.
You should use the format like that https://github.com/vlingo-net/vlingo-net-http/blob/master/src/Vlingo.Http.Tests/vlingo-actors.json

Along with the loader like that https://github.com/vlingo-net/vlingo-net-cluster/blob/master/src/Vlingo.Cluster/Model/Properties.cs

Reply:
Fixed